addons/api/addon/models/role.js

```diff
@@ -33,6 +42,66 @@ export default class RoleModel extends GeneratedRoleModel {
    */
   @attr({ readOnly: true, emptyArrayIfMissing: true }) grant_strings;
 
+  /**
+   * Convenience for looking up the grant scopes, if loaded.
+   */
+  get grantScopes() {
+    const grantScopes = [];
+    if (this.grant_scope_ids) {
+      const sortedScopeIDs = this.grant_scope_ids.slice().sort((a, b) => {
```

Comment:
You might be able to use the new [toSorted](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted) function for this.

Otherwise I think doing 
```suggestion
      const sortedScopeIDs = [...this.grant_scope_ids].sort((a, b) => {
```
is a bit more obvious that you're copying the array first